Content.Client/GameObjects/Components/Mobs/Actions/ActionAssignments.cs

```diff
@@ -93,10 +92,13 @@ public ActionAssignments(byte numHotbars, byte numSlots)
             // which once had an associated item but have been revoked (based on our newly provided action states)
             // so we can dissociate them from the item. If the provided action states do not
             // have a state for this action type + item, we can assume that the action has been revoked for that item.
-            var assignmentsWithoutItem = new List<KeyValuePair<ActionAssignment,List<(byte Hotbar, byte Slot)>>>();
+            var assignmentsWithoutItem = new List<KeyValuePair<ActionAssignment, List<(byte Hotbar, byte Slot)>>>();
             foreach (var assignmentEntry in _assignments)
             {
-                if (!assignmentEntry.Key.TryGetItemActionWithItem(out var actionType, out var item)) continue;
+                if (!assignmentEntry.Key.TryGetItemActionWithItem(out var actionType, out var item))
```

Comment:
Try to avoid unnecessary style changes like this.

It can stay in the PR, just a note for the future.

Reply:
ayy, I really like the explicit if-clause xd but ok